.github/workflows/ci.yml

```diff
@@ -66,6 +66,23 @@ jobs:
           -DS3_PRESIGNED_PUT_URL="put-url"
       - name: Build System Tests
         run: make -C build/ help | grep system | tr -d '. ' | xargs make -C build/
+  build-check-install:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone This Repo
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Configure CMake build
+        run: |
+          curl https://cmake.org/files/v3.2/cmake-3.2.0-Linux-x86_64.tar.gz -o cmake.tar.gz
+          tar -xf cmake.tar.gz
+          mkdir build && cd build
+          ../cmake-3.2.0-Linux-x86_64/bin/cmake .. \
+          -DBUILD_DEMOS=0 -DBUILD_TESTS=0
+      - name: Install libraries
+        run: |
+          make install
```

Comment:
There should be a check for verifying that the libraries and headers have been installed in the target directories. Instead of checking that each of the libraries has been installed, maybe only a handful of libraries can be installed and verified in a separate run of `make install`.
Also, it would be good to test the `CSDK_HEADER_INSTALL_PATH` and `DCSDK_LIB_INSTALL_PATH` variables.